engine/Engine_Phonotype.sc

```diff
@@ -105,6 +116,13 @@ Engine_Phonotype : CroneEngine {
 			PTVoiceAllocator.deliverNoteBend(msg[1].asInt, msg[2].asInt, msg[3].asFloat, msg[4].asFloat);
 		});
 
+		this.addCommand("crow_out", "iff", { arg msg;
```

Comment:
I don't think we need this one; the crow out is handled well by `luaOscAddr.sendMsg(...)` above, and we can do crow in by adding additional control busses to the array of them (and we already have a command to frob control busses)